docs/embeddable-uis/element/access-requests.mdx

```diff
@@ -0,0 +1,589 @@
+---
+sidebar_position: 2
+title: Access Requests API Documentation
+---
+import HelpSupportTile from "@site/src/components/elements/HelpSupportTile.js";
+import ActionContainer from "@site/src/components/elements/ActionContainer.js";
+import ActionTile from "@site/src/components/elements/ActionTile.js";
+import CodeDropdown from "@site/src/components/elements/CodeDropdown.js";
+import FlexWrapper from "@site/src/components/elements/FlexWrapper.js";
+import code from "@site/src/components/elements/codeBlock.json";
+
+# Access Requests API Documentation
+
+This documentation provides details about the Access Requests API.
+
+You can also find information about the Access Requests API in the [Permit Redoc](https://api.permit.io/v2/redoc).
+## Overview
+
+The Access Requests API allows you to manage access requests in your application .
```

Comment:
Remove space between "application" and "."

Reply:
Just because it looks weird - 
![image](https://github.com/permitio/docs/assets/58729960/e5547c3e-0e73-4924-967e-93f3ea386a21)

Reply:
fixed